parts/linux/cloud-init/artifacts/ubuntu/cse_install_ubuntu.sh

```diff
@@ -80,6 +80,13 @@ cleanUpGPUDrivers() {
     rm -Rf $GPU_DEST /opt/gpu
 }
 
+installCriCtlPackage() {
+    version="${1:-}"
+    updateAptWithMicrosoftPkg
```

Comment:
don't we already `updateAptWithMicrosoftPkg` before this?

Reply:
Yes. I will remove it. Thanks for catching this